pooltool/ptmath/utils.py

```diff
@@ -244,7 +244,7 @@ def norm3d(vec: NDArray[np.float64]) -> float:
     2.65 µs ± 63 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)
     241 ns ± 2.57 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
     """
-    return sqrt(vec[0] ** 2 + vec[1] ** 2 + vec[2] ** 2)
+    return sqrt(norm3d_squared(vec))
```

Comment:
I left this out because I wasn't sure if numba would inline the function call. I'm curious what the benchmark results are with this change.

Reply:
I ran the docstring test when I made this change and it ran in 224ns, but I didn't do a proper head to head